xml/System.Linq/Enumerable.xml

```diff
@@ -8257,18 +8257,12 @@ In Visual Basic query expression syntax, an `Aggregate Into Max()` clause transl
         <param name="keySelector">A function to extract the key for each element.</param>
         <summary>Returns the maximum value in a generic sequence according to a specified key selector function.</summary>
         <returns>The value with the maximum key in the sequence.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-If `TKey` is a reference type and the source sequence is empty or contains only values that are `null`, this method returns `null`.
-
-          ]]></format>
-        </remarks>
+        <remarks>If <typeparamref name="TSource" /> is a reference type and the source sequence is empty or contains only values that are <see langword="null" />, this method returns <see langword="null" />. If <typeparamref name="TSource" /> is a primitive type and the source sequence is empty, an <see cref="T:System.InvalidOperationException" /> is thrown.</remarks>
```

Comment:
I would probably rephrase this along the lines of: "If the source enumerable is empty, then two possible outcomes are possible depending on the source type: if the type is nullable, then it returns null, if it is a non-nullable struct then it will throw".